project/MiMa.scala

```diff
@@ -735,7 +735,15 @@ object MiMa extends AutoPlugin {
 
         // #19828
         ProblemFilters.exclude[DirectAbstractMethodProblem]("akka.persistence.Eventsourced#ProcessingState.onWriteMessageComplete"),
-        ProblemFilters.exclude[ReversedAbstractMethodProblem]("akka.persistence.Eventsourced#ProcessingState.onWriteMessageComplete")
+        ProblemFilters.exclude[ReversedAbstractMethodProblem]("akka.persistence.Eventsourced#ProcessingState.onWriteMessageComplete"),
+
+        // #19834 rewrite Utf8Encoder with with GraphStages
+        ProblemFilters.exclude[FinalClassProblem]("akka.http.impl.engine.ws.Utf8Encoder"),
+        ProblemFilters.exclude[DirectMissingMethodProblem]("akka.http.impl.engine.ws.Utf8Encoder.inSurrogatePair"),
+        ProblemFilters.exclude[DirectMissingMethodProblem]("akka.http.impl.engine.ws.Utf8Encoder.onPush"),
+        ProblemFilters.exclude[DirectMissingMethodProblem]("akka.http.impl.engine.ws.Utf8Encoder.surrogateValue_="),
+        ProblemFilters.exclude[DirectMissingMethodProblem]("akka.http.impl.engine.ws.Utf8Encoder.onUpstreamFinish"),
+        ProblemFilters.exclude[DirectMissingMethodProblem]("akka.http.impl.engine.ws.Utf8Encoder.surrogateValue")
```

Comment:
could we try `FilterAnyProblemStartingWith("akka.http.impl") to see if we can reduce the number of merge conflicts?